tests/Fixtures/TestBundle/Entity/PaginationEntity.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the API Platform project.
+ *
+ * (c) Kévin Dunglas <dunglas@gmail.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace ApiPlatform\Tests\Fixtures\TestBundle\Entity;
+
+use ApiPlatform\Metadata\ApiResource;
+use ApiPlatform\Metadata\Get;
+use ApiPlatform\Metadata\GetCollection;
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * ProviderEntity.
+ *
+ * @ORM\Entity
+ */
+#[ApiResource]
+#[Get]
+#[GetCollection(
+    paginationClientItemsPerPage: true,
+    paginationItemsPerPage: 5,
+    paginationMaximumItemsPerPage: 30
+)]
```

Comment:
You cannot use PHP 8 attributes directly in the tests, you need to use annotations like before, they will be automatically converted to PHP attributes by our command (`api:upgrade-resource`).

Reply:
you can you just need to flag them as `@php8` tests :)